src/engine/instrument.h

```diff
@@ -375,6 +385,13 @@ struct DivInstrumentAmiga {
   }
 };
 
+struct DivInstrumentX1_010 {
+  int bankSlot;
```

Comment:
Question! What is this?

Reply:
for Seta 2 style bankswitching support, That is for initial sample bank slot (131072 byte boundary)